packages/suite/src/views/onboarding/steps/Backup/index.tsx

```diff
@@ -139,7 +139,7 @@ const BackupStep = () => {
                         </OnboardingButtonCta>
                     }
                 >
-                    <OptionsWrapper>
+                    <OptionsWrapper fullWidth={false}>
```

Comment:
replacing `flex: 1` with `margin:auto` on StyledImage would do the same thing, but probably it doesn't matter (I just always trying to avoid those configuration props as fullWidth, but here it's already defined...)